network-api/networkapi/templates/fragments/blocks/nav/dropdown.html

```diff
@@ -1,13 +1,13 @@
 {% load static wagtailcore_tags wagtailadmin_tags %}
 
 {% fragment as title_base_typography %}tw-font-sans tw-font-bold tw-text-xl tw-capitalize tw-text-black{% endfragment %}
-{% fragment as content_base_styles %}tw-py-0 tw-px-8 tw-gap-8 tw-overflow-y-auto{% endfragment %}
+{% fragment as content_base_styles %}tw-py-0 tw-px-0 tw-gap-8 tw-overflow-y-auto{% endfragment %}
 {% fragment as dropdown_selector_base %}tw-flex tw-flex-row tw-items-center tw-justify-between tw-w-full tw-py-12 tw-px-8 tw-gap-4{% endfragment %}
 
-{% fragment as content_desktop %}large:tw-container large:tw-hidden{% endfragment %}
+{% fragment as content_desktop %}large:tw-max-w-[1250px] xlarge:tw-max-w-[1350px] 2xl:tw-max-w-[1600px] large:tw-w-full large:tw-px-24 large:tw-hidden{% endfragment %}
```

Comment:
@mmmavis design still needs to finalize which values should go into the `max-width`s here

Reply:
```suggestion
{% fragment as content_desktop %} tw-container large:tw-hidden{% endfragment %}
```

Looks like if we simplify these width and padding classes into just `tw-container`, it would achieve the design we want.

<img width="1469" alt="image" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/2896608/b05cf08e-64f2-4ddd-82e9-aedb4d26bd1f">

Reply:
I had `tw-container` before but the designers wanted it to extend it past the Mozilla logo and the Newsletter button, which the container class won't do :(